indico/MaKaC/webinterface/pages/abstracts.py

```diff
@@ -152,14 +152,6 @@ def getVars(self):
         return wvars
 
 
-class WPConferenceCFA( WPConferenceDefaultDisplayBase ):
```

Comment:
While it doesn't matter much here, we usually don't start removing legacy pages until the very end of a version. To avoid collisions with a legacy blueprint you can always add `-old` to the old URL.

FWIW, you forgot to remove `UHConferenceCFA`, but doing so would require you to change more places where it's currently used.